pytorch_lightning/trainer/trainer.py

```diff
@@ -408,36 +409,13 @@ def __init__(
         # Callback system
         self.on_init_end()
 
-    def fit(
+    def _fit_impl(
         self,
         model: LightningModule,
         train_dataloader: Any = None,
```

Comment:
Should we remove `train_dataloader ` and `val_dataloaders` and `damodule` and `self.train_loop.setup_fit(model, train_dataloader, val_dataloaders, datamodule)` inside of fit function.

Reply:
Would `dispatch_laucher` be a better name than `_fit_impl` ? Or something else. This function does more than fitting.

Reply:
Done both comments. Called it `_launch` instead